### DISCUSSION TITLE
Is Gex_nUMI supposed to be automatically calculated?

### DISCUSSION THREAD

**Discussion by markphillippebworth** [GreenleafLab/ArchR#874] [Questions / Documentation] [answered]
When I add a GeneExpressionMatrix to an ArchR project, is it supposed to automatically calculate the Gex_nUMI metadata column? I'm running it and realizing its not there. And no error was hit when I tried to run addIterativeLSI within the GeneExpressionMatrix, with the depthCol set to Gex_nUMI. The results were terrible though.

**Comment by rcorces** [comment, marked as answer]
`Gex_nUMI` is automatically calculated from your `seRNA`. If its empty, then there is probably something wrong with your `seRNA`
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/MatrixGeneExpression.R#L77

**Reply by markphillippebworth** [reply]
Late response, but I was able to get it to work. Something odd happened the first time (the gene expression matrix was present, but no metadata summaries added). By adding the gene expression matrix again, I was able to get Gex_nUMI and other columns added to the metadata.
